data/json/recipes/food/distill.json

```diff
@@ -0,0 +1,83 @@
+[
+    {
+        "type" : "recipe",
+        "result": "brandy",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_DRINKS",
+        "skill_used": "cooking",
+        "difficulty": 3,
+        "time": 30000,
+        "batch_time_factors": [50, 4],
+        "autolearn": [ [ "cooking", 4 ] ],
+        "book_learn": [ [ "brewing_cookbook", 2 ] ],
+        "qualities": [
+            { "id": "BOIL", "level": 2 },
+            { "id": "DISTILL", "level": 2 }
+        ],
+        "tools": [ [ [ "hotplate", 10 ], [ "char_smoker", 10 ], [ "toolset", 10 ], [ "fire", -1 ] ] ],
+        "components": [ [
+            [ "fruit_wine", 21 ], [ "bum_wine", 21 ],
+            [ "dandelion_wine", 21 ], [ "pine_wine", 21 ],
+            [ "wine_riesling", 15 ], [ "wine_chardonnay", 15 ],
+            [ "wine_cabernet", 15 ], [ "wine_noir", 15 ]
+        ] ]
+    },
+    {
+        "result": "vodka",
+        "type" : "recipe",
+        "id_suffix": "from_wash",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_DRINKS",
+        "skill_used": "cooking",
+        "difficulty": 5,
+        "time": 30000,
+        "batch_time_factors": [50, 4],
+        "autolearn": [ [ "cooking", 6 ] ],
+        "book_learn": [ ["brewing_cookbook", 4] ],
+        "qualities": [
+            { "id": "BOIL", "level": 2 },
+            { "id": "DISTILL", "level": 2 }
+        ],
+        "tools": [
+            [ [ "fire", -1 ], [ "hotplate", 10 ], [ "toolset", 10 ] ]
```

Comment:
Ethanol will probably require more "passes" (or however is it called) due to higher purity.